contracts/truefi2/PoolFactory.sol

```diff
@@ -79,11 +82,16 @@ contract PoolFactory is IPoolFactory, Claimable {
      * @dev Initialize this contract with provided parameters
      * @param _poolImplementationReference First implementation reference of TrueFiPool
      */
-    function initialize(ImplementationReference _poolImplementationReference, ERC20 _stakingToken) external initializer {
+    function initialize(
+        ImplementationReference _poolImplementationReference,
+        ERC20 _stakingToken,
+        ITrueLender2 _trueLender
+    ) external initializer {
         Claimable.initialize(msg.sender);
 
         stakingToken = _stakingToken;
         poolImplementationReference = _poolImplementationReference;
+        trueLender = _trueLender;
     }
 
```

Comment:
Add a function setTrueLender(ITrueLender2 _trueLender2) onlyOwner

This will allow us to change the TrueLender if we ever need to update it.

Reply:
Do we want it? We don't expect TrueLender to be changed and these setters only add points where something can go wrong